src/vstest.console/Resources/xlf/Resources.cs.xlf

```diff
@@ -1683,6 +1683,26 @@
         <target state="new">- {0} {1}</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="EnvironmentArgumentHelp">
+        <source>-e|--Environment|/e|/Environment:&lt;NAME&gt;=&lt;VALUE&gt;
+      Sets the value of an environment variable. Creates the variable if one with the requested name does not exist. This will force the tests to be run in an isolated process.
```

Comment:
Would the behavior be the same as if using /InIsolation switch? Could we say that this implies /InIsolation switch?

Reply:
Wording the sentece as: "Creates the variable if it does not exist." seems to be much easier to understand but convey the same message. It would also be nice to mention that the parameter can be used multiple times to provide multiple variables. And maybe that when the same variable is specified multiple times the last one wins.

And finally it would be nice to document the order in which variables are defined when defined from multiple places, but this is probably not the place to do it.

Reply:
> Would the behavior be the same as if using /InIsolation switch? Could we say that this implies /InIsolation switch?

Yes, we can. Added that to help string.